MatthewKelly12/here_we_grow_django_py
# added node_modules to gitignore, removed node_modules

## Link to Ticket
Closes
 
## Description of Proposed Changes
-added node_modules to .gitignore
-removed node_modules

## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout remove_node_modules
open in editor

```

1. verify file structure


## Impacted Areas in Application

- development environment
